rend3-routine/src/culling/mod.rs

```diff
@@ -65,6 +65,10 @@ pub fn add_culling_to_graph<'node, M: Material>(
 
         let count = graph_data.object_manager.get_objects::<M>(key).len();
 
+        graph_data
+            .get_data(temps, skinned_handle)
+            .expect("Skinning must run before culling");
+
```

Comment:
I'm not sure if we want to keep this. Before, if you forgot to add skinning before culling, it would still run because the skinning output value was not being used. With this it becomes a hard error.

On one hand, it helps catch dumb mistakes like forgetting to add skinning when you create your own render graph, but on the other hand, we may wan to support the use case running culling with no skinning at all?

Reply:
I would say that we want to leave this optional because if people are building a graph and don't do any animation they may want to omit the skinning.